src/index.html

```diff
@@ -8,9 +8,39 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigSearch">
+      <form
+        method="get"
+        data-qa="big"
+        class="bigSearchForm"
+      >
+
```

Comment:
Add spaces between components. It will help other developers to read your code

BAD
```
<div>

  <p>
    some text
  </p>

  <p>

    some text
  </p>

</div>
```

GOOD
```
<div>
  <p>
    some text
  </p>

  <p>
    some text
  </p>
</div>
```